### PR TITLE
Updated with new format specification, grammar checks and formatting.

### PR DESCRIPTION
**Pull request by DaanV2**
@SirLich and I have discussed a couple of points that we felt needed updating. 

- Change capital letters in RP and BP to lowercase.
- Changing programming behavior in the community on how namespaces in component groups / events are more used as a type declaration
- Made identifiers simpler, now either organization or personal username, not both as keeping track of possible 5 different namespaces of each programmer can be more of a pain then useful.
- Recommendation to use subfolder instead of _
- support for entity.bp.json and item.rp.json structure but not preferred.
- animation controller to .controller.json though .ac.json is still accepted, but not preferred.
-

### CODE REVIEW

File: knowledge/style-guide.md [hunk 1]
```diff
@@ -15,94 +15,136 @@ parent: Knowledge
 {:toc}
 </details>
 
-This document will present the officially supported Bedrock-Wiki style-guide for addon-creation. The goal of this guide is to promote best-practices while creating addons, and create a consistent format for everyone to follow.
+This document will present the officially supported Bedrock-Wiki style-guide for addon-creation. The goal of this guide is to promote best-practices
+while creating addons and create a consistent format for everyone to follow.
```
Comment (SirLich) [inline knowledge/style-guide.md:19]: Did a newline get added here?

File: knowledge/style-guide.md [hunk 2]
```diff
@@ -15,94 +15,136 @@ parent: Knowledge
...
 
-The style-guide is a living, breathing document, which will evolve as addon-creation evolves. Please get in touch if you think something needs to be updated ot changed. 
+The style-guide is a living, breathing document, which will evolve as addon-creation evolves. Please get in touch if you think something needs to be
+updated or changed.
```
Comment (SirLich) [inline knowledge/style-guide.md:22]: Another rogue newline?

File: knowledge/style-guide.md [hunk 3]
```diff
@@ -15,94 +15,136 @@ parent: Knowledge
...
-| BP Item              | dragon_tooth.item.bp.json  |
-| RP Item              | dragon_tooth.item.rp.json  |
-| Render Controller    | dragon.render.json         |
-| Loot Table           | dragon.loot.json           |
-| Dragon Saddle        | dragon_saddle.recipe.json  |
-| Spawn Rules          | dragon.spawn.json          |
-| Trade Table          | dragon.trade.json          |
-| Particles            | dragon_magic.particle.json |
-| Texture              | dragon.png                 |
+If multiple options are aviable the prefence go to the first found item, however the second one is recognized as valid as well.
```
Comment (SirLich) [inline knowledge/style-guide.md]: Misspelled aviable

File: knowledge/style-guide.md [hunk 4]
```diff
@@ -15,94 +15,136 @@ parent: Knowledge
...
 
-`minecraft` and `minecon` are reserved. Don't use these.
+- `sapphire:dragon`
+- `sirlich:dragon`
 
-For personal projects, use a convenient version of your player name, and for team projects, use a convenient version of your team name.
+**Where to use name-spaces:** ✔️
 
-When multiple developers work a project together, the namespace should always be shared. If credit is required, use sub-indexing: `sapphire.sirlich:dragon`
+- entities
```
Comment (SirLich) [inline knowledge/style-guide.md:81]: Other required spaces:
 - items
 - blocks

File: knowledge/style-guide.md [hunk 5]
```diff
@@ -15,94 +15,136 @@ parent: Knowledge
...
-Where to use name-spaces:
- - entities
- - particles
- - component-groups
- - events
+**Where to use name-spaces optional:**
 
-When not to use name-spaces:
- - do not include your name-space in any folder path or file-name
+- component-groups
```
Comment (SirLich) [inline knowledge/style-guide.md:88]: Other optional
 - geometry
 - sounds

File: knowledge/style-guide.md (hunk 1 above)
Comment (DaanV2) [reply knowledge/style-guide.md:19]: It move part of the line to line 19, my formatter went off :#

File: knowledge/style-guide.md (hunk 4 above)
Comment (DaanV2) [reply knowledge/style-guide.md:81]: Added

File: knowledge/style-guide.md (hunk 5 above)
Comment (DaanV2) [reply knowledge/style-guide.md:88]: Added

File: knowledge/style-guide.md (hunk 3 above)
Comment (DaanV2) [reply knowledge/style-guide.md]: Of all the things I put through Grammarly, I forgot that one :(

**Comment by DaanV2** [comment]
I have pushed another commit

File: knowledge/style-guide.md (hunk 2 above)
Comment (DaanV2) [reply knowledge/style-guide.md:22]: It move part of the line to line 19, my formatter went off :#

**Review by DaanV2 — COMMENTED** [review]
How did I? - ignore- I accidentifly started a review

**Pull request merged**